packages/factory/factory_constants.tpt2

```diff
@@ -0,0 +1,87 @@
+:import factory macros
+
+; ===== VALID IDS =====
+
+; Everything in this list of the form "item(xx)" is a valid target for
+; automation, and can be assigned to the "target_type" variable in
+; "lanuch factory craft". For example, "item(oven)" indicates that
+; "oven" is a valid item, and you can enter that in the box
+; (without the quotes).
```

Comment:
I might be misunderstanding this code, but it seems like everything here is compile-time only, which means ids are never assigned to global variables at runtime, so how can they be fetched as globals in the `launch factory craft` script?

Reply:
See the new loop in `register factory package`, which replaces the calls to the (deleted) load globals scripts.